components/Brokerage/PackageDetails/index.js

```diff
@@ -3,53 +3,68 @@ import BrokerageHeader from '../BrokerageHeader/BrokerageHeader';
 import { Button, Container, Link } from '../../HackneyDS';
 import PackageUserDetails from '../PackageUserDetails';
 import { useRouter } from 'next/router';
-import ReviewPackageInfo from './ReviewPackageInfo';
+import PackageInfo from './PackageInfo';
 import BrokerageBorderCost from '../BrokerageBorderCost';
 import { currency } from '../../../constants/strings';
 import BrokerageTotalCost from '../BrokerageTotalCost';
-import { BROKER_PACKAGE_ROUTE } from '../../../routes/RouteConstants';
+import { BROKER_PACKAGE_ROUTE, BROKERAGE_HUB_ROUTE } from '../../../routes/RouteConstants';
 import SubmitForApprovalPopup from '../BrokerageSubmitForApprovalPopup/SubmitForApprovalPopup';
+import Breadcrumbs from '../../Breadcrumbs';
 
-export const ReviewPackageDetails = ({
+export const PackageDetails = ({
   userDetails,
   packageInfoItems = [],
   summary = [],
   supplierName,
 }) => {
   const router = useRouter();
-  const packageId = router.query.id;
+  const packageId = router?.query?.id;
   const [isOpenedPopup, setIsOpenedPopup] = useState(false);
 
   const [links] = useState([
     { text: 'Care Package', href: '#care-package' },
     { text: 'Weekly Additional Need', href: '#weekly-additional-need' },
     { text: 'One Off Additional Need', href: '#on-off-additional-need' },
+    { text: 'Funded Nursing Care', href: '#funded-nursing-care' },
     { text: 'Care charges', href: '#care-charges' },
     { text: 'Summary', href: '#summary' },
   ]);
 
+  const [breadcrumbs] = useState([
```

Comment:
This is a constant. No need to keep it in `useState`. Better to move it outside of the component